lib/orbital/api.rb

```diff
@@ -2,6 +2,8 @@ module Killbill #:nodoc:
   module Orbital #:nodoc:
     class PaymentPlugin < ::Killbill::Plugin::ActiveMerchant::PaymentPlugin
 
+      FIVE_MINUTES_AGO = (1 * 300)
```

Comment:
How is the delay used? For CyberSource, it is used as a timeout for the reporting API. But here, the state will never be fixed.

Wouldn't it be easier to simply call [fixPaymentTransactionState](https://github.com/killbill/killbill-api/blob/master/src/main/java/org/killbill/billing/payment/api/AdminPaymentApi.java#L43) instead (from a control plugin for instance)?